ejira-core.el

```diff
@@ -94,11 +94,27 @@ The default value is applicable for:
   :type '(alist :key-type (string :tag "JIRA State")
 		:value-type (number :tag "org-todo")))
 
+(defcustom ejira-user-tag-alist '(("Default" . "ejira_assigned"))
+  "Association list to map assigned users to `org-mode' tags."
+  :group 'ejira
+  :tag "JIRA tag map"
+  :type '(alist :key-type (string :tag "JIRA User Full Name")
+                :value-type (string :tag "Org-Mode Tag")))
+
 (defcustom ejira-org-directory "~/org"
   "Directory to use for storing the org-files generated by ejira."
   :group 'ejira
   :type 'string)
 
+(defcustom ejira-custom-fields-list nil
+  "List of (\"Name\" 'customfield_number &optional 'key) lists.
+key is given if the value is a key/value pair."
+  :group 'ejira
+  :type 'list)
+
+(defcustom ejira-fields-map
```

Comment:
Removing this funky looking definition fixes my crashing on load issue. Does this field have any purpose?

Reply:
No. I was planning for it to have one, removed it after the pull request. Unfortunately, my fork now has two additional custom fields that I had to inject into the code, which makes it unsuitable for a new pull request, and I'm not git-savvy enough to keep what's generic and what's only needed for my case apart.

If you get into trouble with ejira--alist-get, my fork has a version that has worked for all cases I've encountered so far (problems with an additional list layer for customfields). Sorry for the inconvenience.

Reply:
Perhaps it would be best to split off a new branch for this PR?

I used to help out volunteers with git all the time, try using these steps to split off the changes to a new branch and you should be good to go with a fresh PR:

1. Make sure you've committed all outstanding changes before getting started! (verify with <kbd>git status</kbd>)
1. Make sure master is checked out first! <kbd>git checkout master</kbd>
2. Create a new branch from master <kbd>git checkout -b fix/use-accountid</kbd>
3. Rewind time to the fix commit <kbd>git reset --hard ceadc713c16e6feea6d73c58f81b856cfe87fd5c </kbd>
4. Open a text editor and remove your customizations at this point so that only the fix changes are now in the file (verify with <kbd>git diff HEAD^</kbd>)
5. Once satisfied that only the right changes show up in the diff, add them to the fix commit with <kbd>git commit --amend</kbd>
6. Push your new branch back to Github <kbd>git push -u origin fix/use-accountid</kbd>
7. Open the branch's page in Github and submit a new PR based on it

Please feel free to @ me or send out and email if you run into any snags, I'm happy to help!

Reply:
Thanks! I'm going to look into this shortly.

Reply:
Done. I hope the new pull request is ok.